Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceSerializer.cpp

```diff
@@ -81,6 +82,15 @@ namespace AzToolsFramework
                 result.Combine(resultInstances);
             }
 
+            PrefabDomUtils::LinkIdMetadata** linkIdMetadata = context.GetMetadata().Find<PrefabDomUtils::LinkIdMetadata*>();
+            if (linkIdMetadata && *linkIdMetadata)
+            {
+                AZ::ScopedContextPath subPathSource(context, "m_linkId");
+
+                result = ContinueStoringToJsonObjectField(
+                    outputValue, "LinkId", &(instance->m_linkId), &InvalidLinkId, azrtti_typeid<AZ::u64>(), context);
```

Comment:
nit : You can use azrtti_typeid<decltype(instance->m_linkId)>() instead of explicitly specifying the datatype of linkId(AZ::u64) here.  Would be helpful if the type ever changes in the future.